images/provisionvpc_policy.tf

```diff
@@ -0,0 +1,73 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows all of the EC2 actions necessary to
+# provision VPCs in the Images account
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "provisionvpcs_doc" {
```

Comment:
Is this a recommended policy that is described in some external documentation?

Reply:
No, that is the result of trial and error and some Googling.  It would be nice if that was documented somewhere, but I suspect there is no easy one-size-fits-all policy for VPCs.